azurelinuxagent/ga/update.py

```diff
@@ -70,6 +70,7 @@
 AGENT_ERROR_FILE = "error.json"  # File name for agent error record
 AGENT_MANIFEST_FILE = "HandlerManifest.json"
 AGENT_PARTITION_FILE = "partition"
+AGENT_UPDATE_SIGNAL_FILE = "AgentUpgradeSignal"
```

Comment:
Agreed on the renaming in general, though some I think some things are good as is. For example:   `AgentUpdating.json` doesn't make sense since this file is not a json file, its just a simple file which contains a timestamp in it so I'll keep it as  `AgentUpgradeSignal`